File: exs/mundo_1/python/013.py
"""
Desafio 013

Problema: Faça um algoritmo que leia o salário de um funcionário
          e mostre seu novo salário, com 15% de aumento.

Resolução do problema:
"""
salario = float(input('Informe seu sálario: R$'))

salario_aumento = salario + (salario * 15 / 100)

print('Salário final: R${:.2f}'.format(salario_aumento))

File: exs/mundo_1/javascript/013.js
/*
Desafio 013

Problema: Faça um algoritmo que leia o salário de um funcionário 
          e mostre seu novo salário, com 15% de aumento

Resolução do problema:
*/

const input = require('readline-sync').question

let salario = Number(input('Informe seu sálario: R$'))

let salario_final = salario*1.15

console.log(`O salário final é ${salario_final.toFixed(2)}`)

